plot.py

```diff
@@ -0,0 +1,37 @@
+import matplotlib.pyplot as plt
+import pandas as pd
+
+# df = pd.read_csv("test_for_plot.txt", sep="\t")
+# df['mean_rain'] = [0.0, 0.0, 0.0, 0.0, 0.0, 2.0, 11.0, 20.0, 17.0,
+#                    6.0, 0.5, 0.2, 0.2, 0.0, 0.0, 0.0, 0.0, 0.0,
+#                    0.0, 0.0, 0.0, 0.2, 0.1, 0.2]
+
+
+def create_plot(df, parameters):
+    # create plot axes limits
+    max_rain = 10
+    rain_max = df["mean_rain"].max()*1.1
+    if max_rain < rain_max:
+        max_rain = rain_max
+
+    max_temp = df["mean_temp"].max()*1.1
+    min_temp = df["mean_temp"].min() - df["mean_temp"].max()*0.1
+
+    # create plot
+    fig, ax = plt.subplots(figsize=(20, 12))
+    # plt.tick_params(axis='y', which='both', labelleft=False, labelright=True)
+    # fig = plt.figure(figsize=(12, 8))
```

Comment:
Again, commented code, is to be uncommented or removed.

Reply:
Done.